go-controller/.golangci.yml

```diff
@@ -4,6 +4,7 @@ run:
 
   skip-files:
     - ".*\\_test.go$"
+    - pkg/crd/egressfirewall/v1/apis
```

Comment:
This entry should be moved under `skip-dirs`

Reply:
oops sorry corrected